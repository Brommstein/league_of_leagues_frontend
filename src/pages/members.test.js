import React from 'react';
import Members from './members';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('Members Component', () => {

    it('component', () => {
        const membersComponent = shallow(<Members />);
        expect(membersComponent).toMatchSnapshot();
    });
});