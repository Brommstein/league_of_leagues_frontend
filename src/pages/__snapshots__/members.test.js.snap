// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Members Component component 1`] = `ShallowWrapper {}`;
